lib/CMakeLists.txt

```diff
@@ -528,6 +528,8 @@ target_include_directories(volk_obj
     PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
+target_link_libraries(volk_obj PRIVATE cpu_features)
```

Comment:
Sorry for this suggestion, it is apparently only valid with CMake 3.12+, otherwise you get this error as seen at least with the Ubuntu Trusty build which presumably uses and older CMake:
```
CMake Error at lib/CMakeLists.txt:531 (target_link_libraries):

  Object library target "volk_obj" may not link to anything.
```

Instead, this should work:
```
target_include_directories(volk_obj PRIVATE $<TARGET_PROPERTY:cpu_features,INTERFACE_INCLUDE_DIRECTORIES>)
```